draft-ietf-httpbis-no-vary-search.md

```diff
@@ -105,15 +106,16 @@ informative:
 
 --- abstract
 
-This specification defines a proposed HTTP response header field for changing how URL search parameters impact caching.
+This specification defines a proposed HTTP response header field for changing how URI query parameters impact caching.
 
 --- middle
 
 # Introduction
 
-HTTP caching {{HTTP-CACHING}} is based on reusing resources which match across a number of cache keys. One of the most prominent is the presented target URI ({{Section 7.1 of HTTP}}). However, sometimes multiple URLs can represent the same resource. This leads to caches not always being as helpful as they could be: if the cache contains the resource under one URI, but the resource is then requested under another, the cached version will be ignored.
+HTTP caching {{HTTP-CACHING}} is based on reusing resources which match across a number of cache keys. One of the most prominent is the presented target URI ({{Section 7.1 of HTTP}}). However, sometimes multiple URIs can represent the same resource. This leads to caches not always being as helpful as they could be: if the cache contains a response under one URI, but the resource is then requested under another, the cached version will be ignored.
```

Comment:
It's a bit odd to call it "response" and then "resource" immediately after, we should be consistent to avoid confusion.